index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang = "en">
+  <head>
+    <meta charset = "UTF-8" />
+    <meta
+      name = "viewport"
+      content = "width=device-width, initial-scale = 1, minimum-scale = 1"
+    />
+    <link rel = "stylesheet" 
+          href = "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" 
+          integrity = "sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" 
+          crossorigin = "anonymous" 
+    />
+    <link rel = "stylesheet" href = "style.css" />
+    <title>Sunzi Bistro</title>
+  </head>
+  <body>
+    <header>
+        <nav>
+            <div class = "nav" >
+                <img src = "img/logo.jpg" alt = "Sunzi Bistro logo" class = "logo" >
+                <h3> &nbsp; Sunzi <br> &nbsp; Bistro </h3>
+                <ul class = "main-nav" >
+                    <li><a href = "#home" >Home</a></li>
```

Comment:
@Sunzi2598 you don't need spaces between arguments and value. It should look something like this
 ``` class="main-nav" ```

Reply:
Please apply this change everywhere.